### DISCUSSION TITLE
Github Desktop Install dfor windows does not work

### DISCUSSION THREAD

**Discussion by badrul1** [community/community#67188] [New to GitHub]
I am getting this error:
Failed to grab lockfile, will retry: C:\Users\xxxx\AppData\Local\SquirrelTemp\badru\AppData\Local\Temp\.squirrel-lock-D7AAEDBB8FE79C6AFB5A902C051E52299B05A5EC: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\xxxx\AppData\Local\SquirrelTemp\xxxx\AppData\Local\Temp\.squirrel-lock-D7AAEDBB8FE79C6AFB5A902C051E52299B05A5EC'.

I also tried the .msi version but that just seems to fail without any messages.

**Comment by queenofcorgis** [comment]
Hi @badrul1 👋🏼 

My team informs me you opened an Issue in their repo and are working with them there! Best of luck 🍀
